tests/serializers/models.py

```diff
@@ -10,6 +10,7 @@
 from decimal import Decimal
 
 from django.db import models
+from django.contrib.contenttypes.models import ContentType
```

Comment:
Do we have to use `Contenttype` here? It would be nice not to add a contrib dependency to core tests.